app/views/draws/show.html.erb

```diff
@@ -1,3 +1,19 @@
 <h1 class="draw-name"><%= @draw.name %></h1>
+<div class="intent-metrics">
```

Comment:
Hm it could be a partial, not sure I see the motivation just yet. In terms of permissions, I did this intentionally - I'm not sure there's a harm in showing broad metrics to the students participating. Do you think it's a problem?